### PR TITLE
Fix generics due to TestNG and Appium upgrades

### PR DESCRIPTION
**Pull request by mach6**

### CODE REVIEW

**Comment by mach6** [comment]
@sebady let me know if you have any concerns with this one.

File: client/src/main/java/com/paypal/selion/platform/grid/browsercapabilities/MobileDriverFactory.java [hunk 1]
```diff
@@ -98,7 +98,7 @@ private static AndroidDriver createAppiumAndroidInstance(URL url, DesiredCapabil
         }
     }
 
-    private static IOSDriver createAppiumIOSInstance(URL url, DesiredCapabilities capability) {
+    private static IOSDriver<?> createAppiumIOSInstance(URL url, DesiredCapabilities capability) {
```
Comment (sebady) [inline client/src/main/java/com/paypal/selion/platform/grid/browsercapabilities/MobileDriverFactory.java]: The wildcard return type here and for android are the only ones I'm not sure about here.  Technically it's a better practice to more narrowly bound this type if we can. For example AndroidDriver<? extends WebElement>

Comment (mach6) [inline client/src/main/java/com/paypal/selion/platform/grid/browsercapabilities/MobileDriverFactory.java]: Good catch. Will take a look.

**Comment by mach6** [comment]
this is merged

**Pull request closed without merging**